e2e/features/layer_visibility.feature

```diff
@@ -0,0 +1,15 @@
+Feature: PermaplanT planting
+    As a user I want to be able to change the layer visibilities
+
+    Scenario: Successfully change plant layer visibility
+        Given I am on the SUTLayerVisibility map page and I have selected the plant layer
+        When I plant something
+        And I turn the plant layer visibility off
+        Then all plants are invisible
```

Comment:
I think we should be more transparent of what is actually tested (image equal etc.)